Editor/CoreAPI/GameLiftFleetManager.cs

```diff
@@ -100,7 +100,7 @@ private async Task<bool> CreateCustomLocationIfNotExists(string fleetLocation)
             }
         }
 
-        private async Task<string> CreateFleet(ComputeType computeType, string fleetLocation, string fleetName)
+        private async Task<string> CreateAnywhereFleet(ComputeType computeType, string fleetLocation, string fleetName)
```

Comment:
The renaming here is not correct since it accepts a `computeType` parameter, so the intention is it can create either a managed fleet or an anywhere fleet, i would leave this as is

Reply:
Reverted